src/test/java/net/minecraftforge/test/AnimalTameEventTest.java

```diff
@@ -0,0 +1,25 @@
+package net.minecraftforge.test;
+
+import net.minecraft.entity.passive.EntityWolf;
+import net.minecraftforge.event.entity.living.AnimalTameEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@Mod(modid = AnimalTameEventTest.MOD_ID, name = "AnimalTameEvent test mod", version = "1.0")
+@Mod.EventBusSubscriber
```

Comment:
Yeah it just needs to be `static`. Here is that comment: https://github.com/MinecraftForge/MinecraftForge/pull/3686#issuecomment-284681329